qa-niaid.planx-pla.net/manifests/fence/fence-config-public.yaml

```diff
@@ -182,4 +182,19 @@ STORAGE_CREDENTIALS: []
 #       length restrictions on group names.
 GOOGLE_GROUP_PREFIX: 'gen3'
 
+S3_BUCKETS:
+  'niaiddata-data-bucket':
+    cred: 'niaiddata_fence-bot'
+    region: 'us-east-1'
+  'niaidprod-data-bucket':
+    cred: 'niaiddata_fence-bot'
+    region: 'us-east-1'
+  'niaiddev-generic-databucket':
+    cred: 'fence_bot'
+    region: 'us-east-1'
+# `#DATA_UPLOAD_BUCKET` specifies an S3 bucket to which data files are uploaded,
+# using the `/data/upload` endpoint. This must be one of the first keys under
+# `S3_BUCKETS` (since these are the buckets fence has credentials for).
+DATA_UPLOAD_BUCKET: 'niaiddata-data-bucket'
```

Comment:
these look like the prod buckets, doesn't the QA config contain different buckets?

Reply:
Yes they are and I am making the change to QA_BUCKET.